cmd/lotus-bench/main.go

```diff
@@ -303,8 +303,8 @@ func main() {
 
 			epost2 := time.Now()
 
-			if !bytes.Equal(proof1, proof2) {
-				log.Warn("separate epost calls returned different proof values (this might be bad)")
+			if bytes.Equal(proof1, proof2) {
+				log.Warn("separate epost calls returned the same proof values (this might be bad)")
```

Comment:
This takes some randomness as input on the rust side, so it's actually quite wrong if 2 are the same.